### DISCUSSION TITLE
Custom RSS feeds

### DISCUSSION THREAD

**Discussion by SuperFola** [forem/forem#19747] [Feature Requests]
Afaict forem currently can give an RSS feed of the whole platform, e.g. https://dev.to/rss ; you can also get a feed of all the posts from a specific user: https://dev.to/feed/thomasbnt.

It would be pretty cool to get your own feed, from the tags and users you subscribed to, in an RSS feed, something like `/rss/{user}?timeslice=T` with `T` being something like `5months`, `5years`, to return posts from the user subscriptions in `[now-T, now]`.

I don't know if authentication should be handled, as this would allow anyone to access anyone else's feed. However, most (if not all?) RSS clients do not handle a `Basic-Authorization` header ; might be circumvented by passing a query param `token=base64-token`. If so, tokens could expire but that would be quite bad UX to have to regenerate a token every week/month/year to update your RSS feed link in your app(s).

**Comment by michael-tharrington** [comment]
This is a really cool idea. Appreciate you sharing this one with us SuperFola!

**Comment by rothgar** [comment]
I would love the ability to get a feed from users I follow. Right now I have to add them all individually to my reader which makes it more difficult to manage and sync as I'm trying to read more articles on the platform.
